srcpkgs/zeroinstall/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'zeroinstall'
+pkgname=zeroinstall
+version=2.16
+revision=1
+wrksrc="0install-v${version}"
+build_style=gnu-makefile
+make_install_target="install_system"
+make_check_args="test"
+hostmakedepends="gettext opam m4 tar pkg-config which"
+makedepends="libressl-devel libcurl-devel gnupg glib-devel gtk+3-devel"
+depends="gnupg desktop-file-utils hicolor-icon-theme xdg-utils"
+short_desc="Cross-distribution software installation system"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="LGPL-2.1-only"
+homepage="http://0install.net"
+distfiles="https://github.com/0install/0install/releases/download/v${version}/0install-v${version}.tbz"
+checksum=4a040cd8ab3a55a84f8daec48306b45d9c9662860fe003c927ef8b485cdecef3
+nocross="opam is nocross"
```

Comment:
set to `yes`

Reply:
Ι see a lot of templates in the repo that use nocross to tersely specify the reason. I used an inline comment to do the same ;)